config/settings/base.py

```diff
@@ -204,3 +204,6 @@
         integrations=[DjangoIntegration()],
         traces_sample_rate=0,
     )
+
+# Email Verification settings
+EMAIL_VERIFY_TIMEOUT_SECONDS = env("EMAIL_VERIFY_TIMEOUT_SECONDS", default=3600)
```

Comment:
please explicitly set the type, this means that if one environment accidentally has a string for `EMAIL_VERIFY_TIMEOUT_SECONDS` it will raise an error as we'll want to know about it.

```suggestion
EMAIL_VERIFY_TIMEOUT_SECONDS = env.int("EMAIL_VERIFY_TIMEOUT_SECONDS", default=3600)
```